nodestream_akamai/edns/edns.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from ipaddress import IPv4Address, ip_address
+
+from nodestream.pipeline.extractors import Extractor
+
+from ..akamai_utils.edns_client import AkamaiEdnsClient
+
+
+class AkamaiEdnsExtractor(Extractor):
+    def __init__(self, **akamai_client_kwargs) -> None:
+        self.client = AkamaiEdnsClient(**akamai_client_kwargs)
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+    def get_address_format(self, address):
```

Comment:
since this doesn't use `class` resources, should this be a `@staticmethod`?

alternatively, a private top-level utility method might be more idiomatic (something like `_get_address_format(address)`